app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>User profile</h1>
\ No newline at end of file
+<div>
+<%= render partial: "render", users: @users%>
+<div class="bio-container">
+<h2>Bio</h2>
+<p><%= @users[0].bio %></p>
+</div>
+
+<% @posts.each do |post| %>
+<ul class="main-container">
+    <li class="blog">
+    <a href="<%= user_post_path(post.author_id,post.id) %>" class="blog">
+    <h2><%= post.title %></h2>
+    <p><%= post.text %><p>
+    <div class="features">
+    <span>Comments: <%= post.comments.count %></span>
+    <span>Likes: <%= post.likes.count %></span>
+    </div>
+    </a>
+    </li>
+    </ul>
+<% end %>
+
+<% unless @users[0].posts.length <=3 %>
+<button class="see-all">
+<a href="<%= user_posts_path(@users[0])%>">See all posts</a>
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you used the [link_to](https://www.rubyguides.com/2019/05/rails-link_to-method/) method instead of using the regular anchor tag `<a>`.